.github/workflows/backend-libraries/patch-generic.yml

```diff
@@ -0,0 +1,82 @@
+name: Patch
+
+on:
+  workflow_call:
+    inputs:
+      appId:
+        required: true
+        type: string
+      branchRef:
+        description: 'Release branch (format: release-vX.Y.Z)'
+        required: true
+        type: string
+    secrets:
+      APP_PRIVATE_KEY:
```

Comment:
Can you keep inputs and secrets naming consistent with the ones in backend-applications ? Otherwise it feels like there are different values